Azure/azure-cli
# [SQL] `az sql elastic-pool create`: Add support for HighAvailabilityReplica count for HS Elastic pools

**Description**

Add support for HighAvailabilityReplica count for HS Elastic pools
**Testing Guide**

Scenarios tested: 
1. Create EP with default replica count and assert
2. Update EP replica count and assert
3. Create EP with specified replica count and assert
5. Create pooled VLDB without specifying replica count (because not supported to set on db level when pooled) replica count verify match pool's replica count
6. Move pooled VLDB out of pool without specifying replica count (because not supported on db level when pooled) verify keep the replica count of source pool.
7. Get EP should return correct replica count (specifically test GET)
8. Get pooled VLDB should return correct replica count matching pool's specifically test GET)

**History Notes**

`az sql elastic-pool create`: Add ha-replicas parameter to specify number of replicas
`az sql elastic-pool update`: Add ha-replicas parameter to specify number of replicas

---

This checklist is used to make sure that common guidelines for a pull request are followed.
